src/super_gradients/training/metrics/detection_metrics.py

```diff
@@ -123,36 +49,68 @@ def __init__(self, num_cls: int,
         self.components = len(self.component_names)
         self.post_prediction_callback = post_prediction_callback
         self.is_distributed = super_gradients.is_distributed()
-        self.normalize_targets = normalize_targets
+        self.denormalize_targets = not normalize_targets
```

Comment:
This is dirty. Ideally I think it would be great to instead of the format of the targets (xyxy_normalized or cxcywh for instance). This would be great to know how we need to process the targets later on, and to be able to process any format